### PR TITLE
python3Packages.{pygtrie, betacode}: init

### PR DESCRIPTION
**Pull request by kmein**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [x] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Lassulus** [comment]
getting this error for betacode on python2:
```
builder for '/nix/store/7cc9snhy0kjsskb77mb23fm2z8f03rr9-python2.7-betacode-1.0.drv' failed with exit code 1; last 10 log lines:
  Executing setuptoolsBuildPhase
  Traceback (most recent call last):
    File "nix_run_setup", line 8, in <module>
      exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\\r\\n', '\\n'), __file__, 'exec'))
    File "setup.py", line 22, in <module>
      long_description = read('README.rst'),
    File "setup.py", line 14, in read
      with open(os.path.join(os.path.dirname(__file__), fn), encoding='utf-8') as f:
  TypeError: 'encoding' is an invalid keyword argument for this function
  builder for '/nix/store/7cc9snhy0kjsskb77mb23fm2z8f03rr9-python2.7-betacode-1.0.drv' failed with exit code 1
cannot build derivation '/nix/store/shllrgfahfx5di0vpkigkq0s5znwncp5-env.drv': 1 dependencies couldn't be built
```
maybe disable it on python2?

**Comment by kmein** [comment]
> maybe disable it on python2?

done :heavy_check_mark:

File: pkgs/development/python-modules/pygtrie/default.nix [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{ lib, fetchPypi, buildPythonPackage, ... }:
+buildPythonPackage rec {
+  pname = "pygtrie";
+  version = "2.3";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "00x7q4p9r75zdnw3a8vd0d0w0i5l28w408g5bsfl787yv6b1h9i8";
+  };
+  doCheck = false;
```
Comment (Lassulus) [inline pkgs/development/python-modules/pygtrie/default.nix:9]: a comment why its disabled (in the code) is helpful.

File: pkgs/development/python-modules/betacode/default.nix [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  pname = "betacode";
+  version = "1.0";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0s84kd9vblbjz61q7zchx64a6hmdqb4lillna5ryh0g9ij76g6r5";
+  };
+  preBuild = ''echo > README.rst'';
+  disabled = !isPy3k;
+  propagatedBuildInputs = [ pygtrie ];
+  doCheck = false;
```
Comment (Lassulus) [inline pkgs/development/python-modules/betacode/default.nix:12]: a comment why its disabled (in the code) is helpful.

**Pull request merged**